src/profile/forms/SocialLinks.jsx

```diff
@@ -158,14 +158,19 @@ class SocialLinks extends React.Component {
           ),
           editing: (
             <div role="dialog" aria-labelledby="social-links-label">
-              <form onSubmit={this.handleSubmit}>
+              <form id="editing-form" onSubmit={this.handleSubmit}>
```

Comment:
Hmm, adding an `id` solely for a test isn't generally a good idea, since it's not actually used by the code itself and exists only to make the tests function.  I'd recommend finding the form with a `querySelector`, perhaps based on the `aria-labelledby` attribute on the form's parent.  

In general, writing tests that find elements based on accessibility attributes is actually healthy, as it helps ensure our code is annotated in meaningful ways for screen readers, etc.  Using the `aria-labelledby` here is admittedly somewhat arbitrary, but it should be unique, and helps us avoid adding otherwise unnecessary IDs.